cmd/finch/virtual_machine_remove_test.go

```diff
@@ -49,19 +52,28 @@ func TestRemoveVMAction_runAdapter(t *testing.T) {
 				logger.EXPECT().Info(gomock.Any()).AnyTimes()
 			},
 		},
+		{
+			name: "should forcibly remove the instance",
+			cmd:  removeCmd,
+			args: []string{
+				"--force",
+			},
+			mockSvc: func(logger *mocks.Logger, creator *mocks.LimaCmdCreator, ctrl *gomock.Controller) {
+				command := mocks.NewCommand(ctrl)
+				creator.EXPECT().CreateWithoutStdio("remove", "--force", limaInstanceName).Return(command)
+				command.EXPECT().CombinedOutput()
+				logger.EXPECT().Info(gomock.Any()).AnyTimes()
+			},
+		},
 	}
 
+	//nolint:paralleltest // run in serial because tests with different args can interfere with each other
```

Comment:
Could you elaborate more why they interfere with each other? There shouldn't be any state shared by unit tests.

Reply:
Probably related: https://github.com/runfinch/finch/pull/178#discussion_r1083133815

Reply:
race condition of https://github.com/runfinch/finch/pull/178#discussion_r1083129790 😓

Reply:
yeah, this should be addressed by https://github.com/runfinch/finch/pull/178#discussion_r1083145163